frontends/web/src/routes/account/info/guide.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h } from 'preact';
+import { Entry } from '../../../components/guide/entry';
+import { Guide } from '../../../components/guide/guide';
+import { Translate } from '../../../decorators/translate';
+
+interface BitcoinBasedAccountInfoGuideProps {
+    coinName: string;
+    t: Translate;
+}
+
+export function BitcoinBasedAccountInfoGuide({
+    t,
+    coinName,
+}: BitcoinBasedAccountInfoGuideProps) {
+    return (
+        <Guide>
+            <Entry key="guide.accountInfo.xpub" entry={{
```

Comment:
could just be `entry={t('guide.accountInfo.xpub')}` i think

Reply:
Right. Keep in mind link text do not fallback that's why we can't use the shorthand version here https://github.com/digitalbitbox/bitbox-wallet-app/commit/9b43641ee6b379b9c5f080cdc49a733baa6847ad#diff-7ec1c85f063f25e402825aad376083870f9cf75cdc90c3792d73c60902ddec96